rector.php

```diff
@@ -145,4 +146,5 @@
     $services->set(MakeInheritedMethodVisibilitySameAsParentRector::class);
     $services->set(SimplifyEmptyArrayCheckRector::class);
     $services->set(NormalizeNamespaceByPSR4ComposerAutoloadRector::class);
+    $services->set(ThisCallOnStaticMethodToStaticCallRector::class);
```

Comment:
this service is already in php70 config set, which included already when use `LevelSetList::UP_TO_PHP_74`, so it possibly a bug